README.md

```diff
@@ -97,11 +97,13 @@ $ ./ach_to_yaml.sh /path/to/tnnt/source > scoreboard/fixtures/achievements.yaml
 $ python manage.py loaddata achievements conducts sources trophies
 ```
 
-#### Optional: import test data from xlogfile (e.g. `tnnt-2020-eu`)
-`$ python manage.py shell`
-```python
->>> from scripts.test_data import TestData
->>> TestData().import_and_save_from_xlogfile('tnnt-2020-eu')
+### Running and operation
+The pollxlogs command will read xlog data from the sources defined as fixtures and
+imported above. The aggregate command will loop over all games and compute various
```

Comment:
nit: this seems to imply the xlog data comes only from the fixtures, not the actual xlogs.

Reply:
fixed